### PR TITLE
[RDY] Added webhook secret to comment-monitor

### PR DESCRIPTION
**Pull request by geekodour**
This PR adds check for the webhook secret for comment-monitor. This will allow webhook requests coming from GitHub only.

I've tested this with a new cluster, it's working as expected.

### CODE REVIEW

File: tools/commentMonitor/main.go [hunk 1]
```diff
@@ -73,6 +78,13 @@ func main() {
 	if len(cmConfig.eventMap) == 0 {
 		log.Fatalln("eventmap empty")
 	}
+
+	// Get webhook secret.
+	cmConfig.whSecret, err = ioutil.ReadFile(cmConfig.whSecretFilePath)
```
Comment (krasi-georgiev) [inline tools/commentMonitor/main.go]: I think best to read the secret on every request. This way if you change the secret in k8s you won't need to restart the tool.

Comment (geekodour) [reply tools/commentMonitor/main.go]: I added a `/-/reload` endpoint to reload the config file and webhook secret. It's working locally, yet to test it on a running cluster.

Comment (krasi-georgiev) [reply tools/commentMonitor/main.go]: I think a reload endpoint is an overkill at this point, why not just read the file at every request?

Comment (geekodour) [reply tools/commentMonitor/main.go]: With `/-/reload` it is reloading the `eventmap.yml` file as well as the webhook secret file. If we do it with every request then the eventmap and the webhook will be loaded for every `IssueComment` event on `prometheus/prometheus` on GitHub. Will loading both of these files be a good idea on every request? I thought, as `eventmap.yml` is like the configuration file for comment-monitor it'll be a good idea to add `/-/reload` similar to Prometheus. Please let me know if we should read both of the files on every request, I'll add the change.

Comment (krasi-georgiev) [reply tools/commentMonitor/main.go]: yep it is absolutely fine to read those on every request. I did a test recently and the kernel caches non chnaged files in memory so esensitally these are read from memory when not changed.

Comment (geekodour) [reply tools/commentMonitor/main.go]: @krasi-georgiev added the change.

**Comment by krasi-georgiev** [comment]
yep , good to go, Thanks!

**Pull request merged**